scopes/react/react-native/react-native.env.ts

```diff
@@ -0,0 +1,37 @@
+import { Environment, DependenciesEnv } from '@teambit/envs';
+
+export const ReactNativeEnvType = 'react-native';
+
+export class ReactNativeEnv implements Environment, DependenciesEnv {
+
+    getDependencies(){
```

Comment:
this is not the same as what we had before, as it's not merged now with react deps